e2e/utils/_.bash

```diff
@@ -130,30 +133,38 @@ dfx_start() {
             fi
         done
         if ! $has_param; then
-            args+=( "$param_name" "$param_value" )
+            args+=( "$param_name" ${param_value+"$param_value"} )
```

Comment:
I'm having trouble following what's going on here.  I'm sure this works, but could we do this in a more conventional way by deleting the `$#` check above and doing this?
```
if [[ $# == 1 ]]; then
    args += ( "$param_name" )
else
    local param_value=$2
    args += ( "$param_name" "$param_value" )
fi
```